packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -867,6 +867,32 @@ By performing this association, we can invoke the API gateway using the followin
 https://{rest-api-id}-{vpce-id}.execute-api.{region}.amazonaws.com/{stage}
 ```
 
+## Gateway responses
```

Comment:
Add this to the TOC at the top of this file.

Reply:
Added.